travis/android-wait-for-emulator.sh

```diff
@@ -0,0 +1,25 @@
+#!/bin/bash
+
+# Originally written by Ralf Kistner <ralf@embarkmobile.com>, but placed in the public domain
+
+set +e
+
+bootanim=""
+failcounter=0
+timeout_in_sec=1200 # 10 minutes
+
+until [[ "$bootanim" =~ "stopped" ]]; do
+  bootanim=`adb -e shell getprop init.svc.bootanim 2>&1 &`
+  if [[ "$bootanim" =~ "device not found" || "$bootanim" =~ "device offline"
+    || "$bootanim" =~ "running" || "$bootanim" =~  "error: no emulators found" ]]; then
+    let "failcounter += 1"
+    echo "Waiting for emulator to start"
```

Comment:
We should make this also output `$bootanim` to get some visibility into what is actually being returned.

Reply:
Ill echo that out if I end up adding back this script